### PR TITLE
ThIssue502 minor updates da p

### PR DESCRIPTION
**Pull request by damienpicard**
These are minor updates that I need for my model. It should be merge after #494 . See issue #502

### CODE REVIEW

**Comment by rubenbaetens** [comment]
These small changes are no longer up-to-date in the new develop.

**Comment by rubenbaetens** [comment]
@damienpicard can you update the #502 branch so this can be merged and a version 0.4 can be made ?

**Comment by damienpicard** [comment]
@rubenbaetens @Mathadon can you have a quick look to these minor changes and merge them if you agree?

File: IDEAS/package.order [hunk 1]
```diff
@@ -1,6 +1,4 @@
 UsersGuide
-PartialSimInfoManager
```
Comment (Mathadon) [inline IDEAS/package.order]: is this correct?

**Comment by Mathadon** [comment]
looks good other than the one comment!

File: IDEAS/package.order (hunk 1 above)
Comment (damienpicard) [inline IDEAS/package.order]: The SimInfoManager is now under BoundaryCondtions

Comment (Mathadon) [inline IDEAS/package.order]: ok!

**Comment by damienpicard** [comment]
@Mathadon @rubenbaetens can be merged I guess? For the new release, let's first see if we can merge #507 too at the same time. All my models are running with this version and it would be nice to push it on the master :)

**Pull request merged**